main.js

```diff
@@ -111,13 +120,15 @@ function showText() {
 inputField.addEventListener('keydown', e => {
   // Add wrong class to input field
   switch (typingMode) {
+		case 'custom':
     case 'wordcount':

```

Comment:
Did you write it this way so that `custom` is handled the same way as `wordcount`? It is hard to tell. Maybe something like this will convey intent better, I'm not really sure.

```suggestion
		case 'custom': case 'wordcount':
```

Reply:
Yes, I did. It's a fall-through and I was going off examples on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch).

Reply:
Yeah it doesn’t look bad when the indentation is correct.